networks/moonshadow.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Moonshadow
+description: An overview of the current configuration of the Moonshadow TestNet and information on how to start building on it using Solidity.
+---
+
+# Moonshadow - Moonbeam on Westend
+
+_Updated June 2nd, 2021_
+
+## Goal
+
+Moonshadow is the Moonbeam deployment to the [Westend TestNet](https://polkadot.network/westend-introducing-a-new-testnet-for-polkadot-and-kusama/). It will serve as a dry run for the future network launch process of [Moonriver](https://moonbeam.network/networks/moonriver/launch/)/Moonbeam. Compared to Moonbase Alpha, where we have control over the relay chain, Moonshadow replicates the production environment of Kusama/Polkadot, where the relay chain is an independent blockchain.
+
+## Initial Configuration
+
+Moonshadow has the following configuration:
+
+ - Moonshadow runs as a parachain connected to the Westend relay chain
+ - The parachain has four collators (hosted by PureStake) that are collating blocks
+ - There are two RPC endpoints (hosted by PureStake). People can run full nodes to access their own private RPC endpoints
+
+![Moonshadow Diagram](/images/moonshadow/moonshadow-diagram-v1.png)
+
+## Features
+
+Moonshadow will have the same set of features as Moonbase Alpha. To see the full list, you can visit the [Moonbase Alpha page](/networks/moonbase/#features).
+
+However, as it serves as a dry run for the future network launch process of [Moonriver](https://moonbeam.network/networks/moonriver/launch/)/Moonbeam, it will have some [limitations](#limitations)
```

Comment:
```suggestion
However, as it serves as a dry run for the future network launch process of [Moonriver](https://moonbeam.network/networks/moonriver/launch/)/Moonbeam, it will have some [limitations](#limitations).
```